src/test/java/com/github/dockerjava/api/model/BindTest.java

```diff
@@ -62,4 +98,19 @@ public void toStringDefaultAccessMode() {
         assertEquals(Bind.parse("/host:/container").toString(), "/host:/container:rw");
     }
 
+    @Test
+    public void toStringReadOnlySEL() {
+        assertEquals(Bind.parse("/host:/container:ro,Z").toString(), "/host:/container:ro,Z");
```

Comment:
I am not the fun of hancrest. Single call to assertEquals(s1,s2) is much readable and understandable than calls to 3 DSL functions, sorry.